frappe/desk/doctype/workspace/workspace.py

```diff
@@ -56,8 +56,7 @@ def get_link_groups(self):
 		for link in self.links:
 			link = link.as_dict()
 			if link.type == "Card Break":
-
-				if card_links:
+				if card_links and (not current_card.only_for or current_card.only_for == frappe.get_system_settings('country')): 
```

Comment:
@deepeshgarg007 @GangaManoj Was this tested? 


Dictionaries don't have `only_for` attributes.